storage/innobase/dict/dict0mem.cc

```diff
@@ -260,6 +267,35 @@ dict_add_col_name(
 	return(res);
 }
 
+/* fake col default for recovery */
+UNIV_INTERN
+void
+dict_mem_table_fake_col_default(
+	dict_table_t*           table, /*!< in: table */
+	dict_col_t*             col,	/*!< in: col*/
+	mem_heap_t*             heap	/*!< in: mem_heap for default value */
+)
+{
```

Comment:
Write a function comment that clearly explains the purpose of this function.
The parameters look like they are not `in` only, but they are modified. In which way? The function comment should explain that in high-level terms.